packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -118,23 +119,30 @@ class SnackBar extends StatelessWidget {
         },
         child: new Semantics(
           container: true,
-          child: new Material(
-            elevation: 6,
-            color: _kSnackBackground,
-            child: new Container(
-              margin: const EdgeInsets.symmetric(horizontal: _kSideMargins),
-              child: new Theme(
-                data: new ThemeData(
-                  brightness: ThemeBrightness.dark,
-                  accentColor: theme.accentColor,
-                  accentColorBrightness: theme.accentColorBrightness,
-                  textTheme: Typography.white
-                ),
-                child: new FadeTransition(
-                  opacity: fadeAnimation,
-                  child: new Row(
-                    children: children,
-                    crossAxisAlignment: CrossAxisAlignment.center
+          child: new Dismissable(
+            key: new Key('dismissable'),
+            direction: DismissDirection.down,
+            onDismissed: (DismissDirection direction) {
```

Comment:
dismissable is going to try to shrink the width after the snack bar has slid down. That seems a bit weird. Maybe we should have a variant of dismissable that doesn't do that?